### ISSUE TITLE
Telegram desktop stays with previous quiz in quizbot

### ISSUE THREAD

**Issue by bsde1234** [telegramdesktop/tdesktop#7996]
While quizbot running when  new quiz comes telegram desktop not showing the new one even though it is in bottom. We need to scroll down to view it. It stays with previous quiz. It happens occasionally. Hope you understand.

**Comment by RememberTheAir** [comment]
A bit unclear. Can you rephrase it or provide a screen recording? By the way, you completely got rid of the issue template - there's a reason why it is a thing. You should provide some clear steps to reproduce and your current setup (os and tdesktop version)

**Comment by bsde1234** [comment]
OS: UBUNTU
Telegram Desktop Version: 2.1.7

![quiz](https://user-images.githubusercontent.com/35421561/83855126-47ec5f80-a735-11ea-9ef6-93ced0a8ccdf.jpg)

**Comment by RememberTheAir** [comment]
If you stay at the bottom of the chat, the chat messages will automatically shift up when a new message arrives, and you will see the new message at the bottom. If you scroll up, the app rightfully assumes you're reading some previous message and of course will not bring you back to the bottom when a new message arrives. It's intended to work like that on any chat

**Comment by bsde1234** [comment]
I always stay at bottom of chat, I confirmed it several time.
